spec/spec_helper.rb

```diff
@@ -43,52 +43,54 @@
   # inherited by the metadata hash of host groups and examples, rather than
   # triggering implicit auto-inclusion in groups with matching metadata.
   config.shared_context_metadata_behavior = :apply_to_host_groups
-end
-
-# The settings below are suggested to provide a good initial experience
-# with RSpec, but feel free to customize to your heart's content.
-begin
-  # This allows you to limit a spec run to individual examples or groups
-  # you care about by tagging them with `:focus` metadata. When nothing
-  # is tagged with `:focus`, all examples get run. RSpec also provides
-  # aliases for `it`, `describe`, and `context` that include `:focus`
-  # metadata: `fit`, `fdescribe` and `fcontext`, respectively.
-  config.filter_run_when_matching :focus
```

Comment:
* I get this error when i try to run your tests with this command `rspec` kindly fix it 
![image](https://user-images.githubusercontent.com/52497246/191894897-5176cdd5-b4cc-4a0b-9925-1015cd0f1f22.png)

Reply:
<img width="793" alt="Screenshot 2022-09-23 at 14 29 20" src="https://user-images.githubusercontent.com/40231363/191951401-7a3dcd78-b4dd-44eb-8954-1a54cb0712ff.png">
Hey @Davidosky007 the tests run on my end. Strange!